client/dashboard/sites/overview/site-preview-card.tsx

```diff
@@ -36,6 +36,7 @@ const SitePreviewCard = ( { site }: { site: Site } ) => {
 						bottom: 0,
 						left: 0,
 						overflow: 'hidden',
+						backgroundColor: 'var(--wp-components-color-gray-100)',
```

Comment:
Same here, this feels like it should be a `--dashboard__` variable.

If you don't think it should be customisable per-product then you could always put it in `app/style.scss` instead of `app-dotcom/style.scss`. But it seems like it belongs in the same place as `--dashboard__background-color`.